Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -629,9 +630,22 @@ def update_alert_analyst_verdict_request(self, alert_ids, action):
         response = self._http_request(method='POST', url_suffix=endpoint_url, json_data=payload)
         return response.get('data', {})
 
+    def _create_filter_dict(self, filter_dict):
+        return {
+            filter_key: filter_value
+            for filter_key, filter_value in filter_dict.items()
+            if len(filter_value) != 0
```

Comment:
```suggestion
            if filter_value
```
this would look more cleaner, wouldn't it?
As you are checking if the `filter_value` exists or not